topics/ui/components/tool_window.md

```diff
@@ -6,50 +6,86 @@
 
 ## When to use
 
 Use the tool window to show a large amount of information that the user needs while working side-by-side with the editor or other tool windows. For example, viewing the project structure, running and debugging an application, and viewing git log.
 
-If the amount of information is small, show it in the editor, main toolbar, status bar or popup. For example, show version control changes on the editor gutter, breadcrumbs in the main toolbar, current branch and file details in the status bar, quick doc in a popup.
+## When not to use
 
-If the user does not need the information from the main window to complete an operation, show the content in the popup or dialog window. For example, Search Everywhere popup, Settings dialog.
+### Small amount of information
 
-## Tool window structure
+If the amount of information is small, show it in the editor, main toolbar, status bar, or popup. For example, show version control changes in the editor gutter, breadcrumbs in the main toolbar, current branch and file details in the status bar, quick doc in a popup.
+
+<!--For more information about choosing the correct placement for content, read [UI guidelines on placing content](content_placement.md). //TODO: Write a guideline on placing content in IDE.-->
+
+### Main window content is not needed
 
-![](tool_window_structure.png){width=483}
+If the user does not need the information from the main window to complete an operation, show the content in the popup or dialog window. For example, the <control>Search Everywhere</control> popup, <control>Settings</control> dialog.
+
+## Tool window structure
 
-## How to use
+![](toolwindow_structure.png){width=706}
 
-Give the tool window a name:
+### Tool window name
 
 * The name should be [short and descriptive](writing_short.md), preferably not longer than two words.
 * Use title-case capitalization.
 
-Show the tool window name in the tool window button and header. If there are tabs in the tool window header, add a colon after the tool window name. Otherwise, do not add it.
+<!--Show the tool window name in the tool window button and header. If there are tabs in the tool window header, add a colon after the tool window name. Otherwise, do not add it.-->
+
+![](toolwindow_name.png){width=706}
+
+### Icon
+
+Add an icon for the tool window button. The icon should be created in two sizes: 20×20px and 16×16px, grey, and monochromatic. See how to create icons in the [icons guidelines](icons_style.md).
+
+<!--### Icon
+
+Follow [tool window icons guidelines](icons_style#NNN.md) to create correct icons for tool windows.
+// TODO: publish this chapter when the guideline about icons is public too
+-->
+
+#### Feedback
+
+If the content changes and can contain different types of feedback, for example, new updates or errors, a colored badge is shown above the icon.
+
+![](toolwindow_badge.png){width=706}
+
+Do not change the icon for the tool window button when the content changes.
```

Comment:
I'd suggest moving this sentence above the pic. The space under the pic is small making it less clear to what text the pic belongs.